lib/fx/configuration.rb

```diff
@@ -31,8 +32,16 @@ class Configuration
     # @return Fx adapter
     attr_accessor :database
 
+    # Configuration flag that prioritizes the order in the schema.rb of triggers and functions before other statements
```

Comment:
I follow 80 characters for line wrapping, I'd appreciate if this did so too.

Reply:
Perfect, agree on that, saw online one line comments but did not knew if it was because of less than 80 chars comments or because you wanted them to be in one line